.github/workflows/prerelease.yml

```diff
@@ -9,25 +9,20 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.9.7'
+  GO_VERSION: '1.16'
   INTEGRATION: "nginx"
   ORIGINAL_REPO_NAME: ${{ github.event.repository.full_name }}
   REPO_FULL_NAME: ${{ github.event.repository.full_name }}
   TAG: ${{ github.event.release.tag_name }}
 
 jobs:
-  validate:
-    name: Validate code via linters
+  golangci-lint:
+    name: Validate code via golangci
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
-          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
-      - name: Validate code
-        run: make ci/validate
```

Comment:
i understood we were going to keep this and tackle it in a feature iteration

Reply:
I did it here, but we could do it along with the static analysis iteration